scripts/install-openfaas.sh

```diff
@@ -9,10 +9,10 @@ helm upgrade openfaas --install openfaas/openfaas \
     --set functionNamespace=openfaas-fn \
     --set ingress.enabled=true \
     --set gateway.scaleFromZero=true \
-    --set gateway.readTimeout=300s \
-    --set gateway.writeTimeout=300s \
-    --set gateway.upstreamTimeout=295s \
-    --set faasnetesd.readTimeout=300s \
-    --set faasnetesd.writeTimeout=300s \
+    --set gateway.readTimeout=15m \
+    --set gateway.writeTimeout=15m \
+    --set gateway.upstreamTimeout=14m55s \
+    --set faasnetesd.readTimeout=15m \
```

Comment:
faas-netes should only be doing services calls, sos this should be able to stay at the original 5m which is still too big probably.

Reply:
Done